integration-tests/kubernetes/src/it/kubernetes-build-and-deploy/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Do we really need the full index.html in the tests?

Reply:
No we don't I'll remove them.